automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/SignInPage.scala

```diff
@@ -69,7 +70,10 @@ class GoogleSignInPopup(implicit webDriver: WebDriver) extends WebBrowser with W
   def awaitLoaded(): GoogleSignInPopup = {
 
     val popupTrial = Try {
-      await condition  (id("identifierLink").findElement.isDefined || (id("identifierId").findElement.isDefined))
+      val wait = new WebDriverWait(webDriver,10)
+      wait.until(ExpectedConditions.or(
+        ExpectedConditions.visibilityOfElementLocated(id("identifierLink").by),
+        ExpectedConditions.elementToBeClickable(id("identifierId").by)))
```

Comment:
wait until one condition is true

Reply:
This can be achieved with:
```
await condition (id("identifierLink").findElement.exists(_.isDisplayed) || id("identifierId").findElement.exists(_.isEnabled))(10)
```
Using `await condition` has the advantage that it protects against `StaleElementReferenceException` which has a tendency to happen in our app because of re-renders from react. Of course, this case is the Google pop-up so that probably doesn't apply, but `await condition` is also more concise than manually using `WebDriverWait` and `ExpectedConditions` as is more consistent with the rest of the test code.

I dug into the implementations of `visibilityOfElementLocated` and `elementToBeClickable` and they use `isDisplayed` and `isEnabled` respectively so I am confident that the behavior of my suggested code is the same.

Reply:
Also, it looks like this is the real meaningful change in this PR that should help with test reliability, correct? Without seeing it live, I could believe that being more specific about what to look for (visible and clickable instead of just existing) would help so that looks good to me. 👍